middle_end/flambda2/naming/name_occurrences.ml

```diff
@@ -108,10 +108,12 @@ end = struct
       t land 0xfffff
 
     let num_occurrences_in_types t =
-      (t land 0xfffff_00000) lsr 20
+      (* The constant is computed to avoid problems when the host system
+         is 32 bit. *)
+      (t land (0xfffff lsl 20)) lsr 20
 
     let num_occurrences_phantom t =
-      (t land 0xfffff_00000_00000) lsr 40
+      (t land (0xfffff lsl 40)) lsr 40
```

Comment:
On 32-bit targets, `0xfffff lsl 40` is 0, so the result of the expression in this line is always 0, is that intended?